Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/DeselectionHelper.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine.UIElements;
+
+namespace UnityEngine.InputSystem.Editor
+{
+    internal class DeselectionHelper
+    {
+        /// <summary>
+        /// Currently left as a constant to allow turning deselection on/off to be able to remove issues
+        /// related to not having selections more quickly. Also it might be desirable to either support or
+        /// not support deselection (E.g. pressing ESC while having a current selection).
+        /// </summary>
+        private const bool kAllowDeselection = true;
+
+        private int m_SelectedIndex = -1;
+
+        public bool Select(BaseVerticalCollectionView view, int selectedIndex)
+        {
+            if (!kAllowDeselection)
+            {
+                // This is a workaround to prevent deselection in the list or tree-view, e.g. pressing ESC key.
```

Comment:
Following up on this statement to verify that there is no built-in support or guidance for best-practises with UITK team. For now this workaround seem to do the job.

Reply:
Haven't received any feedback on this one so marking it resolved given workaround seem to do the job and will follow up later if needed.